dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -0,0 +1,450 @@
+        // Every single run has to be tied to the same job id.
+
+        final JobDataMap jobDataMap = new JobDataMap();
+        jobDataMap.put(SiteSearchJobImpl.RUN_NOW, Boolean.FALSE.toString());
+        jobDataMap.put(SiteSearchJobImpl.INCREMENTAL, Boolean.TRUE.toString());
+        jobDataMap.put(SiteSearchJobImpl.INDEX_ALIAS, newIndexAlias);
+        jobDataMap.put(SiteSearchJobImpl.JOB_ID, jobId);
+        jobDataMap.put(SiteSearchJobImpl.QUARTZ_JOB_NAME, jobName);
+        jobDataMap.put(SiteSearchJobImpl.INCLUDE_EXCLUDE, "all");
+        jobDataMap.put(SiteSearchJobImpl.LANG_TO_INDEX, new String[]{Long.toString(defaultLang)});
+        jobDataMap.put(SiteSearchJobImpl.INDEX_HOST, site.getIdentifier());
+
+        final JobDetail jobDetail1 = Mockito.mock(JobDetail.class);
+        Mockito.when(jobDetail1.getJobDataMap()).thenReturn(jobDataMap);
+        final JobExecutionContext context1 = Mockito.mock(JobExecutionContext.class);
+        Mockito.when(context1.getJobDetail()).thenReturn(jobDetail1);
+        Mockito.when(context1.getFireTime()).thenReturn(new Date());
+
+        final SiteSearchJobImpl firstRunJob = new SiteSearchJobImpl();
+        firstRunJob.run(context1);
+
+        String generatedBundleId1 = firstRunJob.getBundleId();
+        Assert.assertNotNull(generatedBundleId1);
+        System.out.println(generatedBundleId1);
+
+        recentAudits = siteSearchAuditAPI.findRecentAudits(jobId, 0, 1);
+        Assert.assertFalse(recentAudits.isEmpty());
+        final SiteSearchAudit siteSearchAudit = recentAudits.get(0);
+
+        final SiteSearchResults search1 = siteSearchAPI.search(siteSearchAudit.getIndexName(), "our-page*",0, 10);
+        Assert.assertTrue(search1.getTotalResults() >= 1);
+
+        final File runOnceBundleRoot = BundlerUtil.getBundleRoot(generatedBundleId1, false);
+        Assert.assertTrue(runOnceBundleRoot.exists());
+
+        // At this point we just finished dumping our content into a brand new index through a run once type of job.
+        // From now on we will have to create an incremental job and continuously feed it.
+        // The first time it'll have to create the whole thing since it's the first incremental run.
+
+        final String incrementalJobBundleId = StringUtils.camelCaseLower(jobName);
+
+        final JobDetail jobDetail2 = Mockito.mock(JobDetail.class);
+        Mockito.when(jobDetail2.getJobDataMap()).thenReturn(jobDataMap);
+        final JobExecutionContext context2 = Mockito.mock(JobExecutionContext.class);
+        Mockito.when(context2.getJobDetail()).thenReturn(jobDetail2);
+        Mockito.when(context2.getFireTime()).thenReturn(new Date());
+
+        final SiteSearchJobImpl secondRunJob = new SiteSearchJobImpl();
+        secondRunJob.run(context2);
+
+        generatedBundleId1 = secondRunJob.getBundleId();
+        Assert.assertEquals(incrementalJobBundleId, generatedBundleId1);
+        System.out.println(generatedBundleId1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093862)

Reply:
removed